src/language-html/printer-html.js

```diff
@@ -671,7 +671,9 @@ function printOpeningTag(path, options, print) {
            */
           (node.isSelfClosing &&
             needsToBorrowLastChildClosingTagEndMarker(node.parent))
-            ? ""
+            ? node.isSelfClosing
+              ? " "
+              : ""
```

Comment:
`node.isSelfClosing` seems to always be truthy here, see the line 672.

Reply:
Another branch above L664